website/docs/r/binding.html.markdown

```diff
@@ -72,8 +71,6 @@ The following arguments are supported:
 
 * `destination_type` - (Required) The type of destination (queue or exchange).
 
-* `properties_key` - (Required) A unique key to refer to the binding.
```

Comment:
Instead of being removed, this should be moved to the `Attribute Reference` section and listed as an exported attribute.